packages/core/src/node/process/raw-process.ts

```diff
@@ -24,6 +25,7 @@ export type RawProcessFactory = (options: RawProcessOptions) => RawProcess;
 @injectable()
 export class RawProcess extends Process {
 
+    id: number;
```

Comment:
should be enough to declare it in `Process`

Reply:
yes